### PR TITLE
B/add axelar pool

### PR DESCRIPTION
**Pull request by memacius32**
Adds axlUSDC/USDC-USDT-USDN stable pool.

-Tested Adding liq and swaping.
- Fix detection of tokens elegible for stable swap. Before it used token names, which may not work if a user adds an address manually. Now it uses token address

Screen Shot 2022-10-05 at 12 09 40

### CODE REVIEW

**Comment by memacius32** [comment]
Updated with imported Token from Token repo.

Screen Shot 2022-10-21 at 12 18 05

**Review by specialist0x — APPROVED** [review]

File: src/state/stableswap/constants.ts [hunk 1]
```diff
@@ -208,6 +212,29 @@ export const STABLESWAP_POOLS: StableSwapPools = {
...
+  [StableSwapPoolName.AXLUSD_USDC_USDT_USN]: {
+    name: StableSwapPoolName.AXLUSD_USDC_USDT_USN,
+    friendlyName: 'axlUSDC-USDC/USDT/USN',
+    lpToken: new Token(
+      ChainId.AURORA,
+      '0x62A0C1Bb3ff819d7ad8B1d69C2270f473aeF6f55',
+      18,
+      'axlUSDC-USDC/USDT/USN TLP',
+      'Trisolaris axlUSDC-USDC/USDT/USN'
+    ),
```
Comment (specialist0x) [inline src/state/stableswap/constants.ts:225]: Maybe move this to `dev/generate_tokens/base_tokens_map.js`  so it's variablized and the token name is consistent across the codebase

File: src/state/stableswap/constants.ts [hunk 2]
```diff
@@ -208,6 +212,29 @@ export const STABLESWAP_POOLS: StableSwapPools = {
...
+    poolTokens: [AXLUSDC[ChainId.AURORA], USDC[ChainId.AURORA], USDT[ChainId.AURORA], USN[ChainId.AURORA]],
+    address: '0xc6618d97b96187Cc06C61A32964f2f5D8690aDda', // MetaSwap
+    metaSwapAddresses: '0x7825C8FD39f0173A019e6fE4440E1BD9C25BA12c', // MetaSwapDeposit
+    type: StableSwapPoolTypes.USD,
+    route: 'usd',
+    underlyingPoolTokens: [
+      AXLUSDC[ChainId.AURORA],
+      new Token(ChainId.AURORA, '0x87BCC091d0A7F9352728100268Ac8D25729113bB', 18, 'USD TLP', 'USDC/USDT/USN')
```
Comment (specialist0x) [inline src/state/stableswap/constants.ts:233]: Same as above

**Comment by memacius32** [comment]
@specialist0x  Closing since we are not going to do this metapool in the end.

Taking into account suggestions for next pool! :)

**Pull request closed without merging**